plugins/prefixIds.js

```diff
@@ -0,0 +1,151 @@
+'use strict';
+
+exports.type = 'full';
+
+exports.active = true;
+
+exports.params = {
+    delim: '__'
+};
+
+exports.description = 'prefix IDs';
+
+
+var path      = require('path'),
+    csstree   = require('css-tree'),
```

Comment:
those 3 deps are really not necessary

just put the regex you need, and for tree walking it's just 
 ```js
function walk(node, cb) {
  cb(node);
  if (node.content) node.content.forEach(walk)
}
```